test/membrane/super.spec.js

```diff
@@ -0,0 +1,32 @@
+import createSecureEnvironment from '../../lib/browser-realm.js'
+
+it('super.value reads should return value from current sandbox', () => {
+  let exported
+  const exportValue = (arg) => {
+    exported = arg;
+  }
+
+  const secureEvalOne = createSecureEnvironment(undefined, { exportValue });
+  secureEvalOne(
+    `class Foo { 
+        constructor() { 
+            this.value = "base" ;
+        } 
+    }
+    exportValue(Foo);`,
+  )
+  const Foo = exported;
+
+  const secureEvalTwo = createSecureEnvironment(undefined, { expect, Foo });
+  secureEvalTwo(`
+    class Bar extends Foo {
+        constructor() {
+            super();
+            super.value = 'bar';
+            expect(super.value).toBe('bar');
```

Comment:
The issue here is that even `this.value` of `"base"` is not respected, right?
Could this be simplified to:
```js
class Bar extends Foo {
  constructor() {
    super();
    expect(this.value).toBe('base');
  }
}
```

Reply:
On further review this test seems invalid. It should be:
We should be testing `this.value` not `super.value` to see if the value set in `super()` is preserved (it appears to be working as intended).

I cannot get:
```js
super();
super.value = 'bar';
super.value // to be anything other than `undefined` outside of the sandbox too
```